cellprofiler/pipeline.py

```diff
@@ -1174,12 +1175,13 @@ def rl():
                     if len(a.split(":")) != 2:
                         raise ValueError("Invalid attribute string: %s" % a)
                     attribute, value = a.split(":")
-                    value = value
-                    value = eval(value)
+                    if attribute in skip_attributes:
+                        continue
+                    # En/decode needed to read example cppipe format
+                    # TODO: remove en/decode when example cppipe no longer has \x__ characters
```

Comment:
Sorry, this is a bug I introduced in trying to carve out some of the home-brewed UTF-16 encoding/decoding :) 
I don't think this should stop your PR from being merged, but I think it might be worth trying this with an example pipeline that _doesn't_ have those escape characters. This would be a new "version" of the pipeline file (you can see in this file I think how the different versions are handled) but I think the quicker we can get away from that the better!

Reply:
@AetherUnbound @bethac07 @0x00b1 would you happen to know where I could get such a pipeline? If not, I could always edit one manually

Reply:
Most of the ones here should work https://github.com/CellProfiler/examples

Reply:
@AetherUnbound After creating a "cleaned" version of ExampleFly.cppipe (opened in python interpreter, ran `file.encode().decode('unicode_escape')`, and saved it as another cppipe file), this branch's Cellprofiler appears to read it successfully (v3.1.8 is able to read the cleaned version as well). That said, I had to hack around to get images to load (doesn't appear to be an issue with the cppipe file but the threading which populates the pipelinecontroller's pathlist).

Hack Steps:
1. loaded the welcome screen example pipeline for the images
2. deleted a couple modules to make sure next step works
3. file->imported cleaned cppipe
4. checked that deleted modules reappeared
5. 'Start test mode'
6. was able to step through a few modules (some still need fixing)

Reply:
What about the other example pipelines?

Reply:
Glad the read in worked! Next step is to get it to write out without that garbage encoding stuff!

Reply:
@bethac07 haven't gotten to around that yet but it's the next thing on my to-do list
@AetherUnbound good catch, that slipped my mind actually

thanks for the feedback!  👍

Reply:
@bethac07 after importing all the cppipe files in the example repo, a common issue that came up was how python 3 treats int divided (/) by int. It resolves into a float whereas // resolves the result as an int. I have incorporated the fixes in Commit https://github.com/CellProfiler/CellProfiler/pull/3766/commits/1c257ae0b6bebf590621822d16cc67f10befb3b8 in this branch.

Reply:
@AetherUnbound I believe it no longer writes out the cppipe with the encoding stuff. Please take a look at PR https://github.com/CellProfiler/CellProfiler/pull/3768.